ProcessMaker/screen-builder
# Bugfix/FOUR-5139: It is not possible to add a record in a loop (4.2)

## Issue & Reproduction Steps

1. Create Screen

2. Add loop with this configuration: 
<img width="267" alt="Screen Shot 2022-01-25 at 4 55 34 AM" src="https://user-images.githubusercontent.com/33489982/150954315-153165a9-4c4d-4544-a6b1-7480bfdd3c61.png">

3. Add a line input inside a loop

4. Create a calc with the next code: 
<img width="795" alt="Screen Shot 2022-01-25 at 4 56 50 AM" src="https://user-images.githubusercontent.com/33489982/150954478-2a60eddb-d6ed-4041-9400-e1b569a45521.png">

5. Press preview button

6. Put the following JSON in data input:

```
{
    "loop_1": [
      {
        "form_input_1": "2473"
      }
    ]
}
```

7. Try to add a new record in a loop

Expected behavior: 
By clicking the "+" or "Add Row" button in the loop, an additional line input would be added.

Actual behavior: 
Clicking the "+" does nothing in the current configuration but changing the data input does allow another line input to be added.

## Solution
- The get() method logic used by the Proxy object in the calculated property needed adjusting to properly return the desired data (if any) within src/mixins/computedProperties.js.

## How to Test
You should be able to follow the repoduction steps after this PR and successfully add rows by using the "+" or "Add Row" button to the loop. Same goes for the "-" or "Delete Row" button.

## Related Tickets & Packages
- [Ticket FOUR-5139](https://processmaker.atlassian.net/browse/FOUR-5139)
- The edits I've made here build on changes @nolanpro and I made in a [previous PR](https://github.com/ProcessMaker/screen-builder/pull/1133). I re-tested with that PR's reproduction steps and everything worked successfully. This PR references [ticket FOUR-2797](https://processmaker.atlassian.net/browse/FOUR-2797).

## Code Review Checklist
